package.json

```diff
@@ -79,7 +79,7 @@
     "dispensary": "0.10.5",
     "es6-promisify": "5.0.0",
     "eslint": "3.19.0",
-    "eslint-plugin-no-unsafe-innerhtml": "EnTeQuAk/eslint-plugin-no-unsafe-innerhtml#4b6b606d50",
+    "eslint-plugin-no-unsafe-innerhtml": "1.0.16",
```

Comment:
This is now called `eslint-plugin-no-unsanitized` but it could have breaking changes so it would make sense to handle it separately.